src/plugins/regions.ts

```diff
@@ -402,6 +402,7 @@ class RegionsPlugin extends BasePlugin<RegionsPluginEvents, RegionsPluginOptions
 
   private initRegionsContainer(): HTMLElement {
     const div = document.createElement('div')
+    div.setAttribute('part', 'regions')
```

Comment:
What use case do you have in mind for this?

Reply:
I wouldn’t add code for “just in case”. It’s all rendered in a shadow DOM for a reason. Other elements selectively have the part attribute for external styling.